mongomock/aggregate.py

```diff
@@ -314,7 +315,29 @@ def _handle_string_operator(self, operator, values):
             "Although '%s' is a valid string operator for the aggregation "
             'pipeline, it is currently not implemented  in Mongomock.' % operator)
 
-    def _handle_date_operator(self, operator, values):
+    def _handle_date_operator(self, operator, values):      
+        if operator == "$dateToString":
+            if isinstance(values, dict):
+                valid_values_keys = ["format","date"]
+                if all(k in values.keys() for k in valid_values_keys):
+                    date=self._parse_basic_expression(values["date"])
+                    if not isinstance(values["format"], str):
+                        raise OperationFailure('Format specifier must be a string')
+                    if "%L" in values["format"]:
+                        raise NotImplementedError('Although %L is a valid format specifier '
+                            'it is currently not implemented in Mongomock.')
+                    invalid_format_specifier = re.search("%[^dGHjLmMSuUVwY%]", values["format"])
+                    if invalid_format_specifier is not None:
+                        raise OperationFailure('Invalid format character %s in format string' % invalid_format_specifier.group())
+
+                    warnings.warn("mongomock: date format output might be different from mongo date format output")
```

Comment:
That's not really OK for me. I suggest you only add the date formats that are the same (or at least the ones you want)